test/plugins/scenarios/go-redisv9/config/excepted.yml

```diff
@@ -18,6 +18,99 @@ segmentItems:
   - serviceName: go-redisv9
     segmentSize: ge 1
     segments:
+      - segmentId: not null
+        spans:
+          - operationName: redis/ping
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Cache
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5014
+            isError: false
+            spanType: Exit
+            peer: redis-server:6379
+            skipAnalysis: false
+            tags:
+              - { key: cache.type, value: redis }
+              - { key: cache.op, value: '' }
+              - { key: cache.cmd, value: ping }
+              - { key: cache.key, value: '' }
+              - { key: cache.args, value: not null }
+      - segmentId: not null
+        spans:
+          - operationName: redis/command
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Cache
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5014
+            isError: false
+            spanType: Exit
+            peer: redis-server:7001
+            skipAnalysis: false
+            tags:
+              - { key: cache.type, value: redis }
+              - { key: cache.op, value: '' }
+              - { key: cache.cmd, value: command }
+              - { key: cache.key, value: '' }
+              - { key: cache.args, value: 'command: map[]' }
+      - segmentId: not null
+        spans:
+          - operationName: redis/cluster slots
```

Comment:
Please don't add the blank in the operation name, they should separate by '/'. Also, whey their have so many no entry segment(only exit span).

Reply:
redis command `cluster slots` is a full operationName. I will handle this action.I find this original PR contributor only use `CreateExitSpan` handle redis hook.